Git.md

```diff
@@ -0,0 +1,80 @@
+https://guides.github.com/introduction/flow/
+
+## Configuration
+
+Configure your global gitingore file, usually in your home directory named `.gitignore_global`. It should contain ignore rules specific for your operating system and editor of choice. You can find these ignore rules [here](https://github.com/github/gitignore/tree/master/Global) and add them to your global gitignore file.
+
+After you have created your global gitignore file update the global git configuration to use your global ignore rules:
+
+`git config --global core.excludesfile ~/.gitignore_global`
+
+
+### _OPTIONAL: Color highlighting_
+
+Depending on your terminal color profile, the default Git diff coloring might not be optimal. The following colors are suggested instead:
+
+```bash
+git config --global color.ui true
+
+git config --global color.diff-highlight.oldNormal    "red bold"
+git config --global color.diff-highlight.oldHighlight "red bold 52"
+git config --global color.diff-highlight.newNormal    "green bold"
+git config --global color.diff-highlight.newHighlight "green bold 22"
+
+git config --global color.diff.meta       "yellow"
+git config --global color.diff.frag       "magenta bold"
+git config --global color.diff.commit     "yellow bold"
+git config --global color.diff.old        "red bold"
+git config --global color.diff.new        "green bold"
+git config --global color.diff.whitespace "red reverse"
+```
+
+## Flow
+
+We use the [Git Flow](https://nvie.com/files/Git-branching-model.pdf) workflow model.
+
+### Master
+The `master` branch is one of the two branches with an infinite lifetime. The source code of `HEAD` always reflects a production-ready state.
+
+### Development
+
+`development` is the second branch with an infinite lifetime. The `development` branch source code of `HEAD` always reflects a state with the latest delivered development changes for the next releases.
+
+When the source code in the `development` branch reaches a stable point and is ready to be released, all of the changes should be merged back into `master` and tagged with a release number.
+
+Usually, an automatic build and deploy to production server hook is triggered everytime a commit is pushed to `master`.
```

Comment:
This depends on the project, but on some projects, the deploy might not be automated and it needs to be triggered manually (from semaphore). This is most commonly done to have better control of when something is deployed so the backend and frontend deploy can be syncronised as necessary.

Reply:
`Usually` it works like that. Deployment specifics can be added in the Deployment chapter